var/spack/repos/builtin/packages/ganglia/package.py

```diff
@@ -1,35 +0,0 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
It probably shouldn't have, but this package being in here surprised me.  If it's unused/unmaintained, leave it, but if there's a maintainer lurking about there have been patches for this for a while, like this one: https://github.com/ganglia/monitor-core/pull/313